### ISSUE TITLE
feature request: some sample chords as buttons

### ISSUE THREAD

**Issue by wolftune** [wilckerson/guitar-webplayer#2]
Although the ideal might be some way to type or choose any arbitrary chord, a great beginner-friendly feature (suggested by Aaron Solomon):

Add some simple common chords from some key such that someone (especially on a laptop without touch-screen) can hit a button that says like "Av7" and a full chord is visually shown and played.

And/or have some chord-types like "v7" as a setting and then whatever single note is clicked on the fretboard, it is used as the root of a full chord. Thus, people could move that chord around in parallel.

For future ideals, could have computer keyboard input such that a set of a few chord types could be set by different keys (like 1=v7 2=^m7 3=vM7). Probably that idea should be its own ticket for later though.
